### PR TITLE
Add `test-tube`/`s` icon/s and improve metadata for `lab` icons

### PR DESCRIPTION
**Pull request by danielbayley**
test-tube

### CODE REVIEW

File: icons/test-tube.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "tube",
```
Comment (karsa-mistmere) [inline icons/test-tube.json:4]: Nice one! We could maybe add some (semi)synonyms for easier finding:

```suggestion
    "tube",
    "vial",
    "phial",
    "flask",
    "ampoule",
    "ampule",
```

File: icons/test-tube.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M21,6L5.4,21.4c-0.8,0.8-2,0.8-2.8,0c-0.8-0.8-0.8-2,0-2.8L18,3" />
+  <path d="M17.1,2L22,6.9" />
+  <line x1="5.5" y1="16" x2="10.5" y2="16" />
```
Comment (karsa-mistmere) [inline icons/test-tube.svg:14]: This is a very basic fix for pixel perfection keeping pretty much the same design:
```suggestion
  <path d="M21 6 5.62 21.38c-.83.83-2.17.83-3 0v0a2.12 2.12 0 0 1 0-3L18 3" />
  <path d="m17 2 5 5" />
  <path d="M15 12H9" />
```

Although I think maybe we could fatten it up by a single unit to make the roundness more pronounced on 100% (and potentially also add a matching pair of vertical test tubes as an alternate):
image

Comment (danielbayley) [reply icons/test-tube.svg]: Yeah, like that idea…

File: icons/test-tube.svg [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M21 6 5.62 21.38c-.83.83-2.17.83-3 0v0a2.12 2.12 0 0 1 0-3L18 3" />
+  <path d="m17 2 5 5" />
+  <path d="M15 12H9" />
```
Comment (karsa-mistmere) [inline icons/test-tube.svg:14]: image
These are changes for the thicker version:
```suggestion
  <path d="M21 7 6.82 21.18a2.83 2.83 0 0 1-3.99-.01v0a2.83 2.83 0 0 1 0-4L17 3" />
  <path d="m16 2 6 6" />
  <path d="M16 12H8" />
```

This is the test-tubes variant:
```svg
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M9 2v17.5A2.5 2.5 0 0 1 6.5 22v0A2.5 2.5 0 0 1 4 19.5V2" />
  <path d="M20 2v17.5a2.5 2.5 0 0 1-2.5 2.5v0a2.5 2.5 0 0 1-2.5-2.5V2" />
  <path d="M3 2h7" />
  <path d="M14 2h7" />
  <path d="M9 11H4" />
  <path d="M20 11h-5" />
</svg>
```

File: icons/beaker.json [hunk 4]
```diff
@@ -1,7 +1,8 @@
 {
   "$schema": "../icon.schema.json",
   "tags": [
-    "cup"
+    "cup",
+    "lab"
```
Comment (karsa-mistmere) [inline icons/beaker.json:5]:
```suggestion
    "lab",
    "chemistry"
```

File: icons/flask-conical-off.json [hunk 5]
```diff
@@ -3,7 +3,8 @@
   "tags": [
     "beaker",
     "erlenmeyer",
-    "non toxic"
+    "non toxic",
+    "lab"
```
Comment (karsa-mistmere) [inline icons/flask-conical-off.json:7]:
```suggestion
    "lab",
    "chemistry"
```

File: icons/flask-conical.json [hunk 6]
```diff
@@ -2,7 +2,8 @@
   "$schema": "../icon.schema.json",
   "tags": [
     "beaker",
-    "erlenmeyer"
+    "erlenmeyer",
+    "lab"
```
Comment (karsa-mistmere) [inline icons/flask-conical.json:6]:
```suggestion
    "lab",
    "chemistry"
```

File: icons/flask-round.json [hunk 7]
```diff
@@ -1,7 +1,8 @@
 {
   "$schema": "../icon.schema.json",
   "tags": [
-    "beaker"
+    "beaker",
+    "lab"
```
Comment (karsa-mistmere) [inline icons/flask-round.json:5]:
```suggestion
    "lab",
    "chemistry"
```

File: icons/pipette.json [hunk 8]
```diff
@@ -2,10 +2,12 @@
   "$schema": "../icon.schema.json",
   "tags": [
     "eye dropper",
-    "color picker"
+    "color picker",
+    "lab"
```
Comment (karsa-mistmere) [inline icons/pipette.json:6]:
```suggestion
    "lab",
    "chemistry"
```

**Review by karsa-mistmere — APPROVED** [review]

**Comment by ericfennis** [comment]
Nice icon.
But I'm not sure about the single test tube designed at an angle.
Looking at the other chemistry glass icons, they are all not angled. So to make this constant, should this one not be angled as well? I kinda like the design tho, so maybe add it as a separate one? Or what do you think?

**Comment by danielbayley** [comment]
> add it as a separate one?

Implemented… Also, I made all the liquid levels consistent.

File: icons/flask-round.svg [hunk 9]
```diff
@@ -13,5 +13,5 @@
   <path d="M14 9.3V1.99" />
   <path d="M8.5 2h7" />
   <path d="M14 9.3a6.5 6.5 0 1 1-4 0" />
-  <path d="M5.58 16.5h12.85" />
+  <path d="M5.5 16h13" />
```
[the remark this answers is not in the document]
Comment (danielbayley) [reply icons/flask-round.svg]: Illustrator keeps outputting `M5.5 16h13`, no matter what I do… So I have just manually dialed it in. I’m using [svg-preview](https://marketplace.visualstudio.com/items?itemName=SimonSiefke.svg-preview) directly in VS Code now to adjust some of these manually.

**Review by ericfennis — APPROVED** [review]
LGTM!

File: icons/flask-round.svg [hunk 10]
```diff
@@ -13,5 +13,5 @@
   <path d="M14 9.3V1.99" />
   <path d="M8.5 2h7" />
   <path d="M14 9.3a6.5 6.5 0 1 1-4 0" />
-  <path d="M5.58 16.5h12.85" />
+  <path d="M5.6 16h12.9" />
```
Comment (karsa-mistmere) [inline icons/flask-round.svg:16]: This matches the flask's body to one 100th of a point:
```suggestion
  <path d="M5.52 16h12.96" />
```

**Review by ericfennis — APPROVED** [review]
LGTM

**Pull request merged**